mono/metadata/Makefile.am

```diff
@@ -89,7 +89,11 @@ if !ENABLE_ILGEN
 ilgen_libraries = libmono-ilgen.la
 endif
 
-noinst_LTLIBRARIES = libmonoruntime-config.la $(boehm_libraries) $(sgen_libraries)
+if !HOST_WIN32
+sysnat_libraries = libmonoruntime-sysnat.la
+endif
```

Comment:
It would be easier to just add them to libmonoruntime, having convenience libs just slows down compilation.

Reply:
We have different CFLAGS for `libmonoruntime-sysnat.la` than for `libmonoruntime.la`.